allennlp/data/tokenizers/word_splitter.py

```diff
@@ -134,6 +135,26 @@ def _remove_spaces(tokens: List[spacy.tokens.Token]) -> List[spacy.tokens.Token]
     return [token for token in tokens if not token.is_space]
 
 
+class WhitespaceTokenizer:
```

Comment:
Looks like this is only for the spacy word splitter?  I'd make it private then, and maybe put it inside the `SpacyWordSplitter` class.  Having it here means it will show up in the docs, which would be confusing.